packages/app/cypress/e2e/cypress-in-cypress-e2e.cy.ts

```diff
@@ -273,4 +273,22 @@ describe('Dom Content', () => {
     cy.get('.passed > .num').should('contain', 1)
     cy.get('.failed > .num').should('contain', '--')
   })
+
+  describe('accessibility', () => {
+    it('has no axe violations in specs list panel', () => {
+      cy.visitApp()
+      cy.contains('withFailure.spec').click()
+      cy.get('[data-cy="spec-duration"]')
```

Comment:
What purpose does `cy.get('[data-cy="spec-duration"]')` serve?

Reply:
I didn't see you updated your original suggestion - I think your update would make this unnecessary. I was getting some flake because the runner panel wasn't always rendered before running axe, so this was an attempt to ensure the render was complete.

Come to think of it, I'm not even making the search panel visible via the extra click on the "Specs" control, so that's a better way anyways